aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3081,13 +3073,12 @@ std::tuple<Tensor, Tensor, Tensor> linalg_svd(const Tensor& self, bool full_matr
   TORCH_CHECK(self.dim() >= 2,
               "svd input should have at least 2 dimensions, but has ", self.dim(), " dimensions instead");
 
-    bool some = !full_matrices;
-    Tensor U, S, V;
-    std::tie(U, S, V) = at::_svd_helper(self, some, /*compute_uv=*/true);
-
-    Tensor Vh = V.conj().transpose(-2, -1);
-    return std::make_tuple(U, S, Vh);
+  bool some = !full_matrices;
+  Tensor U, S, V;
+  std::tie(U, S, V) = at::_svd_helper(self, some, /*compute_uv=*/true);
 
+  Tensor Vh = V.conj().transpose(-2, -1);
+  return std::make_tuple(U, S, Vh);
```

Comment:
No idea why i tchose to go with this terrible diff, but this indents this paragraph to 2 spaces rather than 4, to be consistent with the rest of the file.